dashboard/test/models/pd/payment_term_test.rb

```diff
@@ -0,0 +1,56 @@
+require 'test_helper'
+
+class Pd::PaymentTermTest < ActiveSupport::TestCase
+  freeze_time
+
+  self.use_transactional_test_case = true
+
+  setup_all do
+    @regional_partner_1 = create :regional_partner
+    @regional_partner_2 = create :regional_partner
+
+    @workshop_1 = create :pd_workshop, num_sessions: 1, sessions_from: Date.today, regional_partner: @regional_partner_1
+    @workshop_2 = create :pd_workshop, num_sessions: 1, sessions_from: Date.today + 3.months, regional_partner: @regional_partner_1
+    @workshop_3 = create :pd_workshop, regional_partner: @regional_partner_2
+  end
+
+  test 'raises error if no applicable payment terms' do
+    create :pd_payment_term, regional_partner: @regional_partner_1, start_date: Date.today + 4.months
+    [@workshop_1, @workshop_3].each do |workshop|
```

Comment:
Not that important, but I'm curious why test only workshops 1 and 3, where 2 would also be excluded.